HOW_TO.md

```diff
@@ -120,9 +122,19 @@ Breaking apart this command line, we have the following pieces:
 - `/mnt/images` mounts the sample plot images to the running docker container
 
 Output should be in the format of image name and calculated value for that image on a single line for each of the images
-in the images folder
+in the images folder. Example output from the images in the [Google Drive](https://drive.google.com/file/d/1xWRU0YgK3Y9aUy5TdRxj14gmjLlozGxo/view?usp=sharing) 
```

Comment:
Our standard for MD files, which probably isn't documented anywhere, is to have sentences on individual lines